source/administration/config-settings.rst

```diff
@@ -669,6 +669,15 @@ Port of SMTP email server.
 | This feature's ``config.json`` setting is ``"SMTPPort": ""`` with string input.                                                                                      |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+SMTP Server Timeout
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The maximum amount of time (in seconds) allowed for establishing a TCP connection between Mattermost and the SMTP server, to be idle before being terminated.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"SMTPServerTimeout": 0`` with numerical input.                                                                           |
```

Comment:
The default value for this setting should be greater than 0, we can change this line like this 👇 
```suggestion
| This feature's ``config.json`` setting is ``"SMTPServerTimeout": 10`` with numerical input.                                                                           |
```